components/style/gecko_bindings/structs_debug.rs

```diff
@@ -1176,7 +1175,10 @@ pub mod root {
         pub struct ThreadSafeAutoRefCnt {
             pub mValue: u64,
         }
-        pub const ThreadSafeAutoRefCnt_isThreadSafe: bool = true;
+        extern "C" {
+            #[link_name = "_ZN7mozilla20ThreadSafeAutoRefCnt12isThreadSafeE"]
+            pub static ThreadSafeAutoRefCnt_isThreadSafe: bool;
+        }
```

Comment:
I am using `--features llvm_stable`.  Does it make sense to make regen.sh require to a newer clang?